src/renderer/components/cluster-manager/cluster-manager.tsx

```diff
@@ -35,12 +35,16 @@ import { Extensions, extensionsRoute } from "../+extensions";
 import { HotbarMenu } from "../hotbar/hotbar-menu";
 import { EntitySettings, entitySettingsRoute } from "../+entity-settings";
 import { Welcome, welcomeRoute, welcomeURL } from "../+welcome";
+import { ClusterTopbar } from "./cluster-topbar";
+import { CatalogTopbar } from "./catalog-topbar";
 
 @observer
 export class ClusterManager extends React.Component {
   render() {
     return (
       <div className="ClusterManager">
+        <Route component={CatalogTopbar} {...catalogRoute} />
```

Comment:
This looks a bit odd, why do we route to top bars?

Reply:
If showing TopBar without route differentiation, we will just move it inside `<TopBar/>` component itself... It shows different content (label and close button) for Catalog as well as for Cluster views. So this way we would need parsing current route, match it to `catalogUrl()`, `/cluster/:clusterId` somewhere inside `componentDidMount()`. I found using `<Route/>` more clear and explicit.

Reply:
Ok, for me it's a bit confusing but we can always refactor this later if needed.